### DISCUSSION TITLE
[HELP] Cannot sync

### DISCUSSION THREAD

**Discussion by ghost** [Chia-Network/chia-blockchain#3627] [Q&A]
Hi guys, sice i've updated Chia to 1.1.3 about 2h after release, i am not able to sync. Downgrade to 1.1.2 also did not help. Before 1.1.3 i did not have to make any changes in router/modem because of port 8444, but just for make it save i've made port forwarding and UPnP. In log it looks like a problem with SSL. I am on Windows 10 19042.964 with latest updates and synced time. Have you got any ideas?

**Edit:** I have also updated Python from whatever version i had to latest 3.9.4

![obrazek](https://user-images.githubusercontent.com/22098380/116968342-95aa7880-acb4-11eb-94f0-08cdcda9fd93.png)

[debug.log](https://github.com/Chia-Network/chia-blockchain/files/6419319/debug.log)

**Comment by MarcioZamperetti** [comment]
click in "Connect To Other Peers" and add:
IP: node.chia.net
Port: 8444

**Reply by ghost** [reply]
Well, it does not help. It does not anything at all:

![obrazek](https://user-images.githubusercontent.com/22098380/116971846-7878a880-acba-11eb-8148-c1c02791b622.png)

And from Log:
2021-05-04T09:22:34.348 full_node chia.rpc.rpc_server     : WARNING  Error while handling message: Traceback (most recent call last):
  File "chia\rpc\rpc_server.py", line 212, in safe_handle
  File "chia\rpc\rpc_server.py", line 200, in ws_api
  File "chia\rpc\rpc_server.py", line 161, in open_connection
ValueError: Start client failed, or server is not set

**Comment by DIlbert25** [comment]
> Hi guys, sice i've updated Chia to 1.1.3 about 2h after release, i am not able to sync. Downgrade to 1.1.2 also did not help. Before 1.1.3 i did not have to make any changes in router/modem because of port 8444, but just for make it save i've made port forwarding and UPnP. In log it looks like a problem with SSL. I am on Windows 10 19042.964 with latest updates and synced time. Have you got any ideas?
> 
> **Edit:** I have also updated Python from whatever version i had to latest 3.9.4
> 
> ![obrazek](https://user-images.githubusercontent.com/22098380/116968342-95aa7880-acb4-11eb-94f0-08cdcda9fd93.png)
> 
> [debug.log](https://github.com/Chia-Network/chia-blockchain/files/6419319/debug.log)

After upgrading to version 1.1.3, the system lost all synchronization.



After several attempts:

I reboot, uninstalled Chia, delete the .chia folder,  and reinstalled 1.1.3.

System began to update the nodes slowly, and after 24 hours synchronized.

Now, the wallet, even more slowly, is syncing. The speed will take about 03 to 04 days. Start from 0 and, after 12 hours, are in 50.000

How to increase this wallet synchronization speed?

I managed to open port 8444;
I have the 60 connections "conect to other peers";
I made the inclusion of several connection servers ( node. chia.net..... );
I tried to install, from scratch, on another computer, but the low speed continued.
No error messages in debug.log
